docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -0,0 +1,317 @@
+# Configuring a UAN for K3s (Technical Preview)
+
+**WARNING**: This feature is currently a Technical Preview, as such it requires completion of a prerequisites section. More functionality related to streamlining the configuration and enhancing the experience of rootless podman containers will be added in a future release. This means some of the configuration options may be expected to change.
+
+## UAI Experience on UANs
+In UAN 2.6, a new playbook has been added to create a single node, K3s cluster. The K3s environment will be used to run services necessary to replicate the experience of User Access Instances, UAIs, on one or more UANs.
+
+### Use of K3s
+K3s will serve as the orchestrator of services necessary to replicate the capabilities of UAIs on UAN hardware. This includes HAProxy, MetalLB, and eventually DNS services like ExternalDNS and PowerDNS. Notably, this does **not** orchestrate instances of sshd and podman containers through K3s. As a comparison, K3s and the initial set of services are operating in a similar way to "Broker UAIs" in CSM to handle the SSH ingress and redirection of users into their interactive environment.
+
+### Use of Podman
+Traditional UAIs in CSM required some level of privilege in CSM for access to host volume mounts, networking, and startup activities. Podman offers an attractive solution for an interactive environment in which to place users, they can be rootless containers that do not rely on privilege escalation. When running on UANs, podman containers have access to a hosting environment that is already tailored to users.
+
+### Overview
+The overall component flow for replicating containerized environments for End-Users on UANs is as follows:
+- A user uses `ssh` to initiate a connection the HAProxy load-balancer running in K3s.
+- HAProxy, using the configured load balancing algorithms, will forward the SSH connection to an instance of `sshd`
+- `sshd`, running on a UAN via systemd, will initiate a rootless podman container as the user using the `ForceCommand` configuration.
+- The user is placed in a podman container for an interactive session, or their `SSH_ORIGINAL_COMMAND` is run in the container.
+- When the user disconnects, the podman process exits, and the container is removed.
```

Comment:
I'll change the list format and finish that sentence with a period, but there is limitation on where the SSH connection is forwarded to. It could be the same UAN, a different one, or not a UAN at all.

Reply:
On second thought, I think your suggestion makes sense as the very next line mentions sshd running on a UAN! This document should be more narrowly focussed on that use case.